src/content/docs/en/tutorial/3-components/3.mdx

```diff
@@ -34,7 +34,7 @@ Since your site will be viewed on different devices, let's create a page navigat
     <summary>Show me the code!</summary>
 
     Create a file named `Header.astro` in `src/components/`
-     ```astro title="src/components/Header.astro"
+    ```astro title="src/components/Header.astro"
```

Comment:
I don't know why, but this fixes the indentation of the code.

| before | after |
| --- | --- |
| ![Screenshot 2023-07-15 at 13 47 13](https://github.com/withastro/docs/assets/7889778/a59a3a44-84fd-4b1a-b6df-f483b9b95ab9) | ![Screenshot 2023-07-15 at 13 47 41](https://github.com/withastro/docs/assets/7889778/3af4ab41-2075-4e2c-80e8-4dccc75680a1) |


https://docs.astro.build/en/tutorial/3-components/3/